webapp-django/crashstats/crashstats/jinja2/crashstats/home.html

```diff
@@ -1,5 +1,9 @@
 {% extends "crashstats_base.html" %}
 
+{% block page_title %}
+  Crash Stats Homepage
+{% endblock %}
```

Comment:
This has been messed up for a long while. Figured I'd fix it here since I noticed it while testing.